connector-j-6/src/main/java/com/google/cloud/sql/mysql/SocketFactory.java

```diff
@@ -85,4 +81,17 @@ public Socket beforeHandshake() {
   public Socket afterHandshake() {
     return socket;
   }
+
```

Comment:
Remove this extra blank line. Do you use google-java-format? It's a good idea.

Reply:
Yes, but there are a few other violations so I've avoided using it up until this point. I was planning on adding a check with the check-style plugin as well.

Reply:
I've updated the 4 SocketFactory files, but will push the update to core to a different PR.